bundles/org.openhab.binding.miio/README.md

```diff
@@ -4353,7 +4354,24 @@ e.g. `openhab:send actionCommand 'upd_timer["1498595904821", "on"]'` would enabl
 | power            | Switch  | Power                               |            |
 | humidifierMode   | String  | Humidifier Mode                     |            |
 | humidity         | Number:Dimensionless | Humidity                            |            |
-| setHumidity      | Number  | Humidity Set                        |            |
+| setHumidity      | Number:Dimensionless | Humidity Set                        |            |
+| bright           | Dimmer  | LED Brightness                      |            |
+| buzzer           | Switch  | Buzzer Status                       |            |
+| depth            | Number  | Depth                               |            |
+| dry              | Switch  | Dry                                 |            |
+| usedhours        | Number:Time | Run Time                            |            |
+| motorspeed       | Number  | Motor Speed                         |            |
+| temperature      | Number:Temperature | Temperature                         |            |
+| childlock        | Switch  | Child Lock                          |            |
+
+### Smartmi Evaporative Humidifier (<a name="zhimi-humidifier-cb2">zhimi.humidifier.cb2</a>) Channels
```

Comment:
Markdown should be preferred over HTML.
```suggestion
### Smartmi Evaporative Humidifier ([zhimi.humidifier.cb2](#zhimi-humidifier-cb2)) Channels
```

Reply:
Looks like you suggest to create an link instead.
It should be an anchor... the highest answer suggest to use a html tag, though in the bottom it appears some engines accept creation of a named anchor using { }. Is the engine in OH is using supporting that??
https://stackoverflow.com/questions/5319754/cross-reference-named-anchor-in-markdown

Reply:
Ah, indeed I didn't see that it was supposed to be an anchor. Actually, every headline is an anchor by default and you should be able to link to it with `[text](smartmi-evaporative-humidifier)`.

Reply:
I know, but as many devices have the same name, hence that solution does not work.
hence the anchor goes to the (unique) model string, not the 'friendly' / commercial name